src/integration/test_client.py

```diff
@@ -31,7 +33,19 @@
 from txacme.util import csr_for_names, generate_private_key, tap
 
 
-HOST = u'acme-testing.mithrandi.net'
+@implementer(IReactorTime)
+class BrokenClock(object):
```

Comment:
More seriously: `ImmediateClock` or `ReentrantClock` would be closer to what you mean here, I think.